src/App.js

```diff
@@ -1,13 +1,31 @@
+/*eslint-disable*/
+import {BrowserRouter, Routes, Route, Link} from 'react-router-dom';
+
 import CalculatorUI from './components/CalculatorUI';
 import Quotes from './components/Quotes';
+import Home from './components/Home';
 import './index.css';
 
 function App() {
   return (
-    <div className="app">
-      <Quotes />
-      <CalculatorUI />
-    </div>
+    <BrowserRouter>
+      <div className='app'>
+        <header className='header'>
+          <h1>Math Magcian</h1>
+          <nav>
+            <Link to='/'>Home</Link>
+            <Link to='/calculator'>Calculator</Link>
+            <Link to='/quotes'>Quotes</Link>
+          </nav>
+        </header>
```

Comment:
- [Optional] Kindly, move the `header` part to the different component  to make your code clean 👍